projects/pgai/pgai/cli.py

```diff
@@ -661,7 +661,7 @@ def create(
     from pgai.semantic_catalog.vectorizer import embedding_config_from_dict
 
     d = dict(
-        provider=provider.lower(),
+        implementation=provider.lower(),
```

Comment:
I chose not to rename the option to this function as I think for a layperson, "provider" makes more sense than "implementation".